src/gamecontroller.cs

```diff
@@ -307,16 +353,38 @@ partial interface GCController {
 		[Notification, Field ("GCControllerDidDisconnectNotification")]
 		NSString DidDisconnectNotification { get; }
 
-		[iOS (8,0), Mac (10,10)]
+		[iOS (8,0), Mac (10,10), TV(13,0)]
 		[Export ("motion", ArgumentSemantic.Retain)]
 		GCMotion Motion { get; }
 
 		[iOS (9,0)][Mac (10,11)]
 		[Export ("handlerQueue", ArgumentSemantic.Retain)]
 		DispatchQueue HandlerQueue { get; set; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Export ("productCategory")]
+		string ProductCategory { get; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Export ("snapshot")]
+		bool Snapshot { [Bind ("isSnapshot")] get; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Export ("capture")]
+		GCController Capture { get; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Static]
+		[Export ("controllerWithMicroGamepad")]
+		GCController ControllerWithMicroGamepad { get; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Static]
+		[Export ("controllerWithExtendedGamepad")]
+		GCController ControllerWithExtendedGamepad { get; }
```

Comment:
No verb in method
I think `GetExtendedGamepadController ()` is better

Reply:
Same as above, Right now it is set as a property though and properties don't need a verb correct? I can revert to a method though.

Reply:
yeah, I was not clear - it should be a method (and have a verb)

Reply:
Changing!